lms/static/scripts/frontend_apps/components/SvgIcon.js

```diff
@@ -1,57 +1,116 @@
-import classNames from 'classnames';
```

Comment:
This file was copied from `src/shared/components/svg-icon.js` in the client repo with no changes.